config/schema/social_auth_google.schema.yml

```diff
@@ -2,9 +2,12 @@ social_auth_google.settings:
   type: config_object
```

Comment:
Please, don't just copy/paste other implementers' file. Be aware of what you are doing